target.json

```diff
@@ -43,11 +43,11 @@
             "branch":"master",
             "type":"git"
         },
-        {
-            "name":"codal-mbedos",
-            "url":"https://github.com/lancaster-university/codal-mbedos",
-            "branch":"master",
-            "type":"git"
-        }
+      {
```

Comment:
Same thing, as far as I can tell, nothing changed here. Undo the changes to this file.